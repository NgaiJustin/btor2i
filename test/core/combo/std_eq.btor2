; BTOR description generated by Yosys 0.33+65 (git sha1 90124dce5, clang 15.0.0 -fPIC -Os) for module std_eq.
; ARGS: left=0b1100110011001100 right=0b1111000011110000
1 sort bitvec 32
2 input 1 left ; core.sv:157.34-157.38
3 input 1 right ; core.sv:158.34-158.39
4 sort bitvec 1
5 eq 4 2 3
6 output 5 out ; core.sv:159.18-159.21
; end of yosys output
