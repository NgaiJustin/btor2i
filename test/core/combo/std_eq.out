
Environment:
0 : _
1 : _
2 : 13107
3 : 3855
4 : _
5 : 0
6 : _

out: [0]
